.editorconfig

```diff
@@ -1,4 +1,8 @@
 [*.{kt,kts}]
 # Disabling rules that were added in the latest versions of ktlint
 # tracking here: https://github.com/mozilla-mobile/fenix/issues/4861
-disabled_rules=import-ordering
\ No newline at end of file
+
+ij_kotlin_allow_trailing_comma_on_call_site=true
+ij_kotlin_allow_trailing_comma=true
+
+ktlint_disabled_rules=import-ordering
```

Comment:
Let's add a new line to EOF

Reply:
Do we want to take this opportunity to automatically insert a new line to EOF?

It is as simple as adding` insert_final_newline = true` in this editorconfig file.

Reply:
Yes, let's do that